src/portfft/dispatcher/global_dispatcher.hpp

```diff
@@ -142,6 +146,8 @@ struct committed_descriptor<Scalar, Domain>::calculate_twiddles_struct::inner<de
     }
     std::vector<Scalar> host_memory(static_cast<std::size_t>(mem_required_for_twiddles));
     std::vector<Scalar> scratch_space(static_cast<std::size_t>(mem_required_for_twiddles));
+    LOG_TRACE("Allocating global memory for twiddles for workgroup implementation. Allocation size",
```

Comment:
Maybe. But I also do not like that we already have so many logging-specific cmake variables. And this would add more. So maybe for the future if we need it.